src/ucp/api/ucp.h

```diff
@@ -1105,6 +1107,12 @@ typedef struct ucp_worker_attr {
      * Maximal allowed header size for @ref ucp_am_send_nbx routine
      */
     size_t                max_am_header;
+
+    /**
+     * The name is intended for identification of the worker during tracing and
```

Comment:
No need  for passive form here. The name is the ID of the worker. The tracing tool uses the ID to identity an worker object.

Reply:
Tracing and analysis tools can identify the worker using the name.

The same as above.

Reply:
@rakhmets my main concern with the original documentation is `The name is intended ` which is passive form and hard to parse.